scripting/get5/stats.sp

```diff
@@ -517,8 +517,84 @@ static int GetClutchingClient(int csTeam) {
 public void DumpToFile() {
   char path[PLATFORM_MAX_PATH + 1];
   if (FormatCvarString(g_StatsPathFormatCvar, path, sizeof(path))) {
-    g_StatsKv.ExportToFile(path);
+    DumpToFilePath(path);
+  }
+}
+
+public bool DumpToFilePath(const char[] path) {
```

Comment:
A suggestion:

* Add a generic `IsJSONPath` function  to util.sp
* That implementation can be simplified doing something like: `StrCompare(path[path_length - 5], ".json") == 0` (after validating path_length > 5 still)
* Replace [this](https://github.com/splewis/get5/blob/fae6a4f76eb042cb555cc8b33ed67ebb0d87cb45/scripting/get5/matchconfig.sp#L149) with a call to that function
* Make this function: `return IsJSONPath(path) : DumpToJSONFile(path) :  g_StatsKv.ExportToFile(path);`

Reply:
Implemented in a3e9d2b6323fa5df4fae241eb0753aadf3d4f820